components/Category.js

```diff
@@ -0,0 +1,39 @@
+import styled from "styled-components";
+import { useState } from "react";
+
+import CategoryHeadline from "./CategoryHeadline";
+import CategoryListItem from "./CategoryListItem";
+
+import { exampleListItems } from "../lib/db";
+
+export default function Category({ category }) {
+  const [isExtended, setIsExtended] = useState(true);
+
+  return (
+    <article key={category.id}>
+      <CategoryHeadline
+        extended={isExtended}
+        handleClick={() => setIsExtended((oldIsExtended) => !oldIsExtended)}
+      >
+        {category.name}
+      </CategoryHeadline>
+      {isExtended && (
+        <CategoryList>
+          {exampleListItems.map((listItem) =>
+            listItem.categoryId === category.id ? (
+              <CategoryListItem key={listItem.id}>
+                {listItem.name}
+              </CategoryListItem>
```

Comment:
Instead of getting the exampleListItems directly from the db, you should pass them down as a prop, ideally you can filter the list before passing them down, so you don't have to do it here.